presto-native-execution/presto_cpp/main/PeriodicTaskManager.cpp

```diff
@@ -86,7 +86,10 @@ void PeriodicTaskManager::start() {
 
   VELOX_CHECK_NOT_NULL(taskManager_);
   addTaskStatsTask();
-  addOldTaskCleanupTask();
+
+  if (SystemConfig::instance()->enableOldTaskCleanUp()) {
```

Comment:
@pratyakshsharma what's the problem if we cleanup old tasks after 1 min by default in Spark-on-Prestissimo? Thanks!

Reply:
@xiaoxmeng Not all Presto APIs are implemented by Sapphire, for example heartbeat API. This results in prestissimo to think that a long running task is stale and it gets purged.
Due to the isolated execution of tasks in Sapphire, this feature is not needed at all.